source/common/http/conn_manager_impl.cc

```diff
@@ -1045,7 +1054,31 @@ ConnectionManagerImpl::ActiveStream::commonEncodePrefix(ActiveStreamEncoderFilte
   if (!filter) {
     return encoder_filters_.begin();
   } else {
-    return std::next(filter->entry());
+    if (filter_iteration_start_state == FilterIterationStartState::Can_start_from_current &&
+        (*(filter->entry()))->iterate_from_current_filter_) {
+      // The filter iteration has been stopped for all frame types, and now the iteration continues.
+      // The current filter's encoding callback has not be called. Call it now.
+      return filter->entry();
+    } else {
+      return std::next(filter->entry());
+    }
+  }
+}
+
+std::list<ConnectionManagerImpl::ActiveStreamDecoderFilterPtr>::iterator
+ConnectionManagerImpl::ActiveStream::commonDecodePrefix(
+    ActiveStreamDecoderFilter* filter, FilterIterationStartState filter_iteration_start_state) {
+  if (!filter) {
+    return decoder_filters_.begin();
+  } else {
```

Comment:
early returned already, no need for else statement

Reply:
Fixed. Thanks!